resources/views/admin/pages/product/add.blade.php

```diff
@@ -1,76 +1,16 @@
-                        @foreach ($colors as $color )
-                        <option value="{{$color->id }}">{{$color->name}}</option>
-                        @endforeach
-                    </select>
-                </label>
-                <label class="block mt-4 text-sm">
-                    <span class="text-gray-700 dark:text-gray-400">
-                        Sizes
-                    </span>
-                    <select class="block w-full mt-1 text-sm dark:text-gray-300 dark:border-gray-600 dark:bg-gray-700 form-select focus:border-purple-400 focus:outline-none focus:shadow-outline-purple dark:focus:shadow-outline-gray" name="size">
-                        @foreach ($sizes as $size )
-                        <option value="{{$size->id }}">{{$size->name}}</option>
-                        @endforeach
-                    </select>
-                </label>
-                <div class="mt-4 text-sm">
-                    <span class="text-gray-700 dark:text-gray-400">
-                        Status
-                    </span>
-                    <div class="mt-2">
-                        <label class="inline-flex items-center text-gray-600 dark:text-gray-400">
-                            <input type="radio"
-                            checked 
-                            class="text-purple-600 form-radio focus:border-purple-400 focus:outline-none focus:shadow-outline-purple dark:focus:shadow-outline-gray" 
-                            name="status" 
-                            value="0" 
-                            />
-                            <span class="ml-2">Pending</span>
-                        </label>
-                        <label class="inline-flex items-center ml-6 text-gray-600 dark:text-gray-400">
-                            <input type="radio" class="text-purple-600 form-radio focus:border-purple-400 focus:outline-none focus:shadow-outline-purple dark:focus:shadow-outline-gray" name="status" value="1" />
-                            <span class="ml-2">delivered</span>
-                        </label>
-                    </div>
-                </div>
-                <div class="mt-4">
-                    <button class="px-4 py-2 bt-3 text-sm font-medium leading-5 text-white transition-colors duration-150 bg-purple-600 border border-transparent rounded-lg active:bg-purple-600 hover:bg-purple-700 focus:outline-none focus:shadow-outline-purple">
-                        Save
-                    </button>
+            @include('admin.pages.product.product_add_update')
+            <div class="mt-4">
+                <button class="px-4 py-2 bt-3 text-sm font-medium leading-5 text-white transition-colors duration-150 bg-purple-600 border border-transparent rounded-lg active:bg-purple-600 hover:bg-purple-700 focus:outline-none focus:shadow-outline-purple">
```

Comment:
make new form.blade.php file and combine the add & update blade code into form.blade file.